### DISCUSSION TITLE
Is there any problem with the license to distribute the self-produced PYINSTALLER auxiliary tools?

### DISCUSSION THREAD

**Discussion by MKdays** [pyinstaller/pyinstaller#5416] [PyInstaller] [answered]
I attached an image to show what concept it is. It's a free app for self-produced and doesn't it matter if I distribute it as an exe file to intuitively help Pyinstaller? Currently, I only have it on my PC.

![image](https://user-images.githubusercontent.com/64838226/103170439-92772a00-4887-11eb-9bbe-3e1b3ed1d596.png)

**Comment by bwoodsend** [comment, marked as answer]
No, that's perfectly OK. There are already a few applications which already do that. [auto-py-to-exe](https://pypi.org/project/auto-py-to-exe/) is quite trendy. I'd have thought you'd have to release yours under the same or a compatible licence to our knock-off GPL2 but I see auto-py-to-exe is MIT which definitely isn't so I'm probably wrong.

Let us know if/when you choose to release yours and we'll add a link to it in our docs.

**Reply by MKdays** [reply]
Thank you. I uploaded the first version to my link. PyInstaller is connected to the About menu.
https://github.com/MKdays/PyKooler/

**Reply by bwoodsend** [reply]
Nice. A couple of suggestions:

- Put a license in that repo. Without one, someone can copy your code, add their own license, pretend it was their code all along then sue you for infringement of their copyright - even though you wrote it to begin with! Just copy the MIT license into your repo's root if you want a no nonsence one.
- Git is meant for source code, not binaries. You'd be better to put your Python code in the repo then release binaries using Github Releases.

**Reply by MKdays** [reply]
Thank you for telling me about the obligation to notify me of the license notice. By the way, since the PySide2 I used is LGPL 3v, I think I should apply the same license. I will make sure about this part and distribute it. Thank you for your suggestion.

**Comment by MKdays** [comment]
I tried to comply with LGPL with PySide 2, but the legal interpretation is still ambiguous.
Have no choice but to release it through GPL. I hope it helps someone.
https://github.com/MKdays/PyKooler
![image](https://user-images.githubusercontent.com/64838226/104912754-06f94080-59d0-11eb-9376-47a71cfb86fe.png)
